### DISCUSSION TITLE
Using spatie/macroable 2.0.0 in PHP 8.1

### DISCUSSION THREAD

**Discussion by jskorlol** [spatie/docker#47] [Ideas]
Using spatie/macroable 2.0.0 in PHP 8.1
In PHP 8.1 you have to lower spatie/macroable to 1.0.1.
Can't upgrade to 2.0.0?
Other spatie packages are using 2.0.0

**Comment by inmanturbo** [comment]
@jskorlol See https://github.com/spatie/docker/pull/52 (Merged)
